### PR TITLE
new durathread items

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
:cl:
add: durathread duffelbag
add: durathread toolbelt
add: durathread bandolier
fix: fixed durathread helmet to not look like a reskinned hardhat (haha)
fix: bandoliers were supposed to hold 18 but the max_combined_weightclass was set lower, this has been fixed
/:cl:

durathread stuff wasnt really worth doing and no one would ever bother asking botany to make downgraded armor or fireproof pants, this gives something that the crew might ask for as these items benefit crew more than botany alone rewarding both for interacting as they can sell these

- duffelbag : no slowdown cobby category 2 bag (by the time you get enough durathread you probably have bohs unlocked, but these are more safe to give out, less expensive and less efficient)

- toobelt : extra slots and can fit stuff like rcds, engis usually have more items available than what the can carry, this should help by creating an interaction between the 2 departments

- bandolier : cobby category 2 bandolier between trashy bag and crappy basic bandolier has 2x capacity

not sure if it was supposed to be 18 tho but i bet old coders meant it to be so else there wouldnt be a "18" written there

### CODE REVIEW

File: code/game/objects/items/storage/belt.dm [hunk 1]
```diff
@@ -27,6 +27,47 @@
...
+		/obj/item/holosign_creator/engineering,
+		/obj/item/forcefield_projector,
+		/obj/item/assembly/signaler,
+		/obj/item/lightreplacer,
+		/obj/item/rcd_ammo,
+		/obj/item/construction/rcd,
+		/obj/item/pipe_dispenser,
+		/obj/item/stack/rods,
+		/obj/item/stack/tile/plasteel,
+		/obj/item/grenade
```
Comment (kriskog) [inline code/game/objects/items/storage/belt.dm]: I get that engineers have smart metal foam grenades, but sneaking in a general /obj/item/grenade into a 14 slot toolbelt just makes this a potential grenade belt. Narrow this down or remove it.

**Comment by actioninja** [comment]
do we really need even more storage powercreep?

**Comment by Tlaltecuhtli** [comment]
there are no intermediates between roundstart and lolBoH for storage,
having as engi not pick immersive tools like the holo projector because its a waste of space is bad
and having botany actually help crew is always good

**Comment by Tlaltecuhtli** [comment]
ah fuck

**Comment by ExcessiveUseOfCobblestone** [comment]
> cobby category 2 bandolier 

nani

**Comment by kriskog** [comment]
> 
> 
> > cobby category 2 bandolier
> 
> nani

Yeah this is clearly a cat 3

**Comment by ExcessiveUseOfCobblestone** [comment]
gonna have to nerf it then, too many cat 3 bandoliers.

**Pull request closed without merging**